superduperdb/ext/openai/model.py

```diff
@@ -47,6 +47,8 @@ class _OpenAI(APIModel):
     :param client_kwargs: The kwargs to be passed to OpenAI
     '''
 
+    openai_api_key: t.Optional[str] = None
```

Comment:
we have in client_kwargs?

Reply:
For high-frequency parameters,  making it more convenient for users to use.